scripts/bulk-password-update-export.js

```diff
@@ -0,0 +1,134 @@
+
+process.env.NODE_TLS_REJECT_UNAUTHORIZED=0;
+const rpn = require('request-promise-native');
+const minimist = require('minimist');
+const {promises: fsPromises} = require('fs');
+const { randomInt } = require('crypto');
+const readline = require('readline');
+
+const argv = minimist(process.argv.slice(2), {
+  alias: {
+    'h': 'help'
+  }
+});
+
+if (argv.h|| !argv.url || !argv.user || !argv.password) {
+  console.log(`Change passwords of users in user-password-change.txt to random value, show results on screen 
+
+Usage:
+      node bulk-password-update-export.js -h | --help
+      node bulk-password-update-export.js --url https://localhost --user medic --password adminPass
+
+Options:
+    -h --help     Show this screen.
+    --url         The url for the instance being changed
+    --user        The admin user this operation is run as
+    --password    The password for the admin user 
+
+`);
+  process.exit(0);
+}
+
+let url;
+try {
+  url = new URL('/api/v1/users', argv.url);
+} catch (e) {
+  console.log('Error while creating url', e.message);
+  process.exit(0);
+}
+
+const user = argv.user;
+const password = argv.password;
+
+const options = {
+  uri: url.href,
+  json: true,
+  headers: {
+    'Authorization': 'Basic ' + Buffer.from(`${user}:${password}`).toString('base64')
+  }
+};
+
+// thanks https://stackoverflow.com/a/50890409 !
+const areYouSure = async (userCount) => {
+  const NOTE = ' ****** WARNING ****** \n\nContinuing will log '
+    + userCount
+    + ' users out of the CHT until you provide them with their updated password. \n\n'
+    + 'Do you want to continue? [y/N]\n';
```

Comment:
Looks like a good use for template literals...

```suggestion
  const NOTE = ` ****** WARNING ******\

Continuing will log ${userCount} users out of the CHT until you provide them with their updated password.

Do you want to continue? [y/N]
`;
```